### PR TITLE
CORE-6579: Throw exception for empty alias

### PR DESCRIPTION
**Pull request by yift-r3**

### CODE REVIEW

File: components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/CertificatesRpcOpsImplTest.kt [hunk 1]
```diff
@@ -309,13 +309,26 @@ class CertificatesRpcOpsImplTest {
...
+
         @Test
         fun `no certificates throw an exception`() {
             assertThrows<InvalidInputDataException> {
                 certificatesOps.importCertificateChain("tenant", "alias", emptyList())
             }
         }
 
+        @Test
+        fun `empty alias throw an exception`() {
```
Comment (nikinagy) [inline components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/CertificatesRpcOpsImplTest.kt]: throws

File: components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/CertificatesRpcOpsImplTest.kt [hunk 2]
```diff
@@ -309,13 +309,26 @@ class CertificatesRpcOpsImplTest {
...
+            val certificateText = ClassLoader.getSystemResource("r3.pem").readText()
+            val certificate = mock<HttpFileUpload> {
+                on { content } doReturn certificateText.byteInputStream()
+            }
+
+            assertThrows<InvalidInputDataException> {
+                certificatesOps.importCertificateChain("tenant", "", listOf(certificate))
```
Comment (nikinagy) [inline components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/CertificatesRpcOpsImplTest.kt:328]: can we check if the exception message contains the word 'alias'?

File: components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/KeysRpcOpsImplTest.kt [hunk 3]
```diff
@@ -238,6 +239,18 @@ class KeysRpcOpsImplTest {
...
 
+        @Test
+        fun `generateKeyPair throws exception for empty alias`() {
+            val publicKey = mock<PublicKey> {
+                on { encoded } doReturn byteArrayOf(1, 2, 3)
+            }
+            whenever(cryptoOpsClient.generateKeyPair("tenantId", "CATEGORY", "alias", "scheme")).doReturn(publicKey)
+
+            assertThrows<InvalidInputDataException> {
+                keysOps.generateKeyPair(tenantId = "tenantId", alias = "", hsmCategory = "category", scheme = "scheme")
```
Comment (nikinagy) [inline components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/KeysRpcOpsImplTest.kt:250]: can we check here too if the exception message contains the word 'alias'?

**Review by nikinagy — COMMENTED** [review]
Last comment, will approve after that.

**Review by nikinagy — APPROVED** [review]
Perfect, LGTM.

**Review by charlieR3 — APPROVED** [review]

**Pull request merged**